usagov/vote-gov-drupal
# VOTE-325: Update drupal core to 10.1.4

## Jira ticket

([VOTE-325](https://cm-jira.usa.gov/browse/VOTE-325))

## Description

Update Drupal Core to latest version 10.1.4

## Deployment and testing

### Pre-deploy

None

### Post-deploy

1. run `lando rebuild`
2. then `lando retune` to run composer install.

### QA/Test

1. Login as an admin and confirm that the Drupal version is 10.1.4 when viewing the admin status page.
2. Click through site and confirm that site is still functional.

## Checklist for the Developer




## Checklist for the Peer Reviewers
